content/en/serverless/guide/azure_app_service_linux_sidecar.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Instrument Azure App Service - Linux Containers with severless-init
+kind: guide
+further_reading:
+- link: "/integrations/azure_app_services/"
+  tag: "Documentation"
+  text: "Azure App Service"
+- link: "/integrations/azure_app_service_environment/"
+  tag: "Documentation"
+  text: "Azure App Service Environment"
+---
+
+<div class="alert alert-info">To instrument your Azure App Service containers with <code>serverless-init</code>, see <a href="/serverless/azure_app_services_container">Instrument Azure App Service - Linux Container with serverless-init</a>.</div>
+
+## Overview
+
+This instrumentation method uses Azure's [Sidecar pattern][1] to monitor containerized Linux Azure App Service workloads.
+
+### Prerequisites
+
+- Your Azure App Service application is containerized
+- You are using a programming language [supported by a Datadog tracing library][2]
+- You have a [Datadog API key][3]
+
+## Instrument your application
+
+1. [Integrate the Datadog tracer][11] into your containerized application
+1. [Create your Linux Web App][12]
+1. [Add Datadog environment variables][13] as application settings
+1. [Add the Datadog sidecar][14]
+
+### Integrate the Datadog tracer
+
+After your application is containerized, integrate the Datadog tracer by adding the following lines to the Dockerfile for your main application.
```

Comment:
Didn't we already pre-require them to have a containerized app? I feel like this should say something slightly different. Instrumented?